app/views/cars/show.html.erb

```diff
@@ -32,53 +32,54 @@
       <% else %>
         <%= nil %>
       <% end %>
-      <%= link_to "Back to car list", :root, class:"btn btn-dark desc-buttons p-4"%>
+      <%= link_to "Back to Search", :root, class:"btn btn-dark desc-buttons p-4"%>
```

Comment:
as we will be implementing a search feature in the index, this page should generally be called 'search' to the user?